cli/cmd/user/remove.go

```diff
@@ -54,6 +56,7 @@ func RemoveCmd(cfg *config.Config) *cobra.Command {
 	removeCmd.Flags().StringVar(&cfg.Org, "org", cfg.Org, "Organization")
 	removeCmd.Flags().StringVar(&projectName, "project", "", "Project")
 	removeCmd.Flags().StringVar(&email, "email", "", "Email of the user")
+	removeCmd.Flags().BoolVar(&removeFromProjects, "remove-from-projects", false, "Remove user from projects")
```

Comment:
This should be `true` by default. Maybe flip it to `--keep-project-roles` with a default of `false`? And a description of "Keep roles granted directly on projects in the org"